core/common/soralog.hpp

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <libp2p/log/configurator.hpp>
+#include <libp2p/log/logger.hpp>
+
+namespace fc {
+  inline void libp2pSoralog() {
```

Comment:
Please, add comment that it is a log for libp2p only. I think `core/common/libp2p/` is a better place for it.